docs/agents/sematext-agent/app-agent/spm-monitor-switching-modes.md

```diff
@@ -15,7 +15,7 @@ which you want to change the mode). Set:
     - **SPM\_MONITOR\_JMX\_PARAMS** should be adjusted according to
       instructions in [Java App Agent Standalone](spm-monitor-standalone)
 
-3.  Restart SPM Monitor:
+3.  Restart Sematext Agent:
 
     ``` bash
     sudo /etc/init.d/spm-monitor restart
```

Comment:
Would it make sense to add a task to change this to sematext-agent or at least add a symlink?

Reply:
I think yes. Should I JIRA?

Reply:
I'll take care of the JIRA.

Reply:
Btw, maybe we shouldn't just change this particular instruction in the docs, it wouldn't work for anyone. And knowing how people are slow with agent upgrades...

Reply:
@lujomon 👍 for ticket

>  maybe we shouldn't just change this particular instruction in the docs, it wouldn't work for anyone. And knowing how people are slow with agent upgrades...

It would work for new users :)
At some point we want to change.  So better to start early.  Suggestion: use comments to inform people about the command for older agents.  Like:

\# if you are using an older agent use: service .....
<real, up to date service command here>